test/e2e/storage/vsphere_scale.go

```diff
@@ -147,7 +147,7 @@ var _ = SIGDescribe("vcp at scale [Feature:vsphere] ", func() {
 			scArrays[index] = sc
 		}
 
-		vsp, err := vsphere.GetVSphere()
+		vsp, err := getVSphere(client)
```

Comment:
Please move all files `test/e2e/storage/vsphere_*` into a new package `test/e2e/storage/vsphere/`. The tests that need the vsphere specific code should import the package. The new package should only publically expose the interfaces and methods that are required by tests, everything else in the package should be package private.

Reply:
Hi Saad,

Thanks for the review.
We already have PR open for this change. 
https://github.com/vmware/kubernetes/issues/353

There are few other things also needed like moving logic where ENV vars are read etc.
Please take a look at the PR for more details.

Reply:
Ack. Please prioritize that higher, this code is making the code base messy. CC @jeffvance 
You don't have to move the tests themselves as long as the tests apply to multiple cloud providers or storage backends, but you should have a separate package for any vmware specific code.